ledger/src/blockstore_db.rs

```diff
@@ -2204,10 +2197,14 @@ pub mod tests {
     {
         pub(crate) fn iterator_cf_raw_key(
             &self,
-            iterator_mode: IteratorMode<Vec<u8>>,
-        ) -> DBIterator {
-            let cf = self.handle();
-            self.backend.iterator_cf_raw_key(cf, iterator_mode)
+            iterator_mode: IteratorMode<C::Index>,
+        ) -> impl Iterator<Item = (Box<[u8]>, Box<[u8]>)> + '_ {
+            // The conversion of key back into Box<[u8]> incurs an extra
+            // allocation. However, this is test code and the goal is to
+            // maximize code reuse over efficiency
```

Comment:
I added this comment for the interim, but this duplicate allocation/conversion will go away later.